docs/source/introduction.rst

```diff
@@ -1,16 +1,18 @@
 Introduction
 ============
 
-`Anticipatory action <https://centre.humdata.org/anticipatory-action/>`_
-(AA) means shifting humanitarian response to be deployed ahead of a crisis.
-Knowing when to provide humanitarian aid involves understanding the
-risk that a potential shock may occur. Gaining this understanding often
+Humanitarians are increasingly recognizing that anticipating
+the risk of a disaster can be more effective than traditional
```

Comment:
Need to be careful with saying risk everywhere because I think there's potential confusion with anticipatory action and risk estimation. Like, down below saying to reduce vulnerability and exposure we have to understand the underlying hazard risk is true, but anticipatory action is not about reducing vulnerability or exposure but about acting ahead of a shock. And I don't think what we do is risk estimation but shock prediction. Wondering if it's better to make the package sound more widely applicable (as it does I guess already in the second paragraph?). Here's just an idea, incorporate as you see fit.

> Whether it is through reducing the vulnerability and exposure of affected populations to likely hazards or acting in anticipation of predictable shocks, humanitarians are increasingly recognizing the importance of estimating the risk and predicting the occurrence of shocks and disasters. This requires a wide array of data from historical observations to existing model forecasts to related covariates.
>
> OCHA AnticiPy provides a simple interface for downloading and processing a range of data relevant for anticipatory action and disaster risk reduction in humanitarian response.

Reply:
Yes amazing point, thank you! I've used your text, just changed it a little bit.